DataFormats/L1TGlobal/interface/GlobalAlgBlk.h

```diff
@@ -109,9 +101,12 @@ class GlobalAlgBlk
     //Prescale Column
     int m_preScColumn;
 
+    //Menu UUID
+    int m_menuUUID;
+    int m_firmwareUUID;
    
     std::vector<bool> m_algoDecisionInitial;
-    std::vector<bool> m_algoDecisionPreScaled;
+    std::vector<bool> m_algoDecisionInterm;
```

Comment:
Masks and prescales are applied in the following order;
  GTL = (1) => AlgoBxMask = (2) => Prescale = (3) => FinOR mask = (4) => 
At the moment uGT algorithms decisions are sampled at (2), (3) and (4)
The request was to change the sampling points to (1), (2) and (3)
In order to shift the sampling point, we need to stop using FinOR mask and
apply prescale value calculated with  prescale[ii] * finormask[ii]. (prescale value of 0 is equivalent to mask in uGT)
The firmeware and software need to be updated at the same time.